src/test/23-c8ql-queries-stream.ts

```diff
@@ -122,7 +122,9 @@ describe34("C8QL Stream queries", function() {
 
       let count = 0;
       Promise.all(
-        Array.apply(null, { length: 25 }).map(() => fabric.query(query, opts))
+        Array.apply(null, { length: 25 } as any).map(() =>
```

Comment:
get rid of any

Reply:
The new Typescript compiler will give us an error. We can @ts-ignore this, leave it as any, or do a refactoring of this part of the code. Do you have any other ideas in mind?